automation/src/test/scala/org/broadinstitute/dsp/consent/chains/DataSetChains.scala

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.dsp.consent.chains
+
+import io.gatling.core.Predef._
+import io.gatling.core.structure.ChainBuilder
+import org.broadinstitute.dsp.consent.requests.Requests
+import spray.json._
+import DefaultJsonProtocol._
+import org.broadinstitute.dsp.consent.models.JsonProtocols
+import org.broadinstitute.dsp.consent.models.DataSetModels._
+import org.broadinstitute.dsp.consent.models.DataAccessRequestModels._
+import org.broadinstitute.dsp.consent.services._
+import scala.collection.mutable.ListBuffer
+
+object DataSetChains {
+    def dataSetCatalogPickTwo(additionalHeaders: Map[String, String]): ChainBuilder = {
+        exec(
+            Requests.User.dataSetCatalog(200, "${dacUserId}", additionalHeaders)
+        )
+        .exec { session =>
+            implicit val dataSetFormatJson = JsonProtocols.dataSetFormat
+            implicit val dataSetEntryJson = JsonProtocols.dataSetEntryFormat
+            implicit val dataAccessRequestJson = JsonProtocols.dataAccessRequestDraftFormat
+
+            var dataSetStr = session(Requests.DataSet.dataSetResponse).as[String]
+            val dataSets = dataSetStr.parseJson.convertTo[Seq[DataSet]]
+            
+            var chosenSets = dataSets
```

Comment:
nit: val

Reply:
noted, var changed to val for both of these variables